README.md

```diff
@@ -1,6 +1,43 @@
-# TEMPLATE-base-repo
+# MAPLE (Bill Summarization, Tagging, Explanation)  
+In this project, we generate summaries and category tags for of Massachusetts bills for [MAPLE Platform](https://www.mapletestimony.org/). The goal is to simplify the legal language and content to make it comprehensible for a broader audience (9th-grade comprehension level) by exploring different ML and LLM services.  
 
-# Add Users
-To add yourself to the repository, open a Pull Request modifying `COLLABORATORS`, entering your GitHub username in a newline.
+This repository contains a pipeline from taking bills from Massachusetts legislature, generating summaries and category tags leveraging different the Massachusetts General Law sections, creating a dashboard to display and save the generated texts, to deploying and integrating into MAPLE platform.
 
-All Pull Requests must follow the Pull Request Template, with a title formatted like such `[Project Name]: <Descriptive Title>`
+## Roadmap of Repository Directories
+* [Documentation](https://github.com/vynpt/ml-maple-bill-summarization/tree/dev/Documentation):  
```

Comment:
It would be better to link the deployment instructions (from the demoapp folder) here.